Refactor/remove get involved page

chriskei opened the pull request:
https://github.com/lukesanantonio/firstbyte/issues/53

features:
- information about how to get involved moved to the team page
- white link at the header that will scroll you down to this new section
- get involved page removed from the header

I decided to keep all of the actual files related to the get involved page in case we decide we want to add back in an events section in the future, as well as the routing to the old page, so http://localhost:8000/events/ still works. If we don't want this link to remain alive, I can remove it, but we may want to consider rerouting users to the teams page in case they click on some old link we previously sent them.

lukesanantonio commented on firstbyte/static/firstbyte/style/main.css:
```diff
@@ -240,3 +240,11 @@ hr {
 }
+
+html {
+    scroll-behavior: smooth;
```
IT'S BEAUTIFUL! :tada:

lukesanantonio commented:
@chriskei So the only thing left to do here is remove that events app, and redirect /events/ to /team/. Now might be a good time to change the footer Get Involved link, and remove the events link there as well. Let me know if this makes sense!

chriskei commented:
Hey Luke, let me know if this update addresses the concerns in your comment.

1. "remove that events app" - deleted the entire events directory, wasn't sure if this was exactly what you intended
2. "redirect /events/ to /team/" - added to urls.py
3. "change the footer Get Involved link" - there is no 'Get Involved' link in the footer, just 'Contact us' and 'Join our team,' so I didn't do anything to address this one
4. "remove the events link there" - done

Let me know if there are more changes that need to be made!

lukesanantonio commented on firstbyte/urls.py:
```diff
@@ -40,5 +40,5 @@
     path('post-survey/', teacher_post_survey),
     path('', include('curriculum.urls')),
-    path('', include('events.urls')),
+    path('events/', team, name='team')
```
Leave out `name='team'` here since that should be reserved for the `team/` endpoint. The name here is only used to retrieve the url from a template: aka `{% url 'team' %}` resolves to `team/`.
